Automattic/wp-calypso
# Add Material icons package and svgr

#### Changes proposed in this Pull Request

Add a package for storing the Material icons we're using in the new nav drawer work (see PR #31742).  The official material-design-icon package (https://github.com/google/material-design-icons) has not been updated since Jan 2018.  It is missing several icons we intend to use.

To load these icons, I'm using svgr, which is also under consideration in core/Gutenberg (https://github.com/WordPress/gutenberg/issues/14628).  This loads the SVG with webpack and transforms it into a React component.

The only place where we load SVGs currently is `client/lib/flags`, which required a minor update to continue working as expected.

This isolates the dependency changes from the remainder of the nav drawer work, which hopefully will aid review and ease merging.

#### Testing instructions



* With these changes, you should be able to load an SVG and get a React component.  `import { ReactComponent as SvgTest } from 'material-design-icons/action/outline-offline_bolt-24px.svg';`.
* The flag tests should still pass.
